### PR TITLE
Task/wp 151 exception form changes

### PR DESCRIPTION
**Pull request by sophia-massie**
## Overview

Added file type to each threshold exception block that dynamically populates that block

## Related

- [WP-151](https://jira.tacc.utexas.edu/browse/WP-151)

## Changes

Added file type to each threshold exception block

## Testing

1. In exception/views.py, add the following snippet on line 184

<details>

cdls = [('CDLME001', 'Data Submitter Code', '100'),
 ('CDLME004', 'Member\nInsurance/Product\nCategory Code', '90'),
 ('CDLME005', 'Start Year of\nSubmission ', '100'),
 ('CDLME006', 'Start Month\nof Submission ', '100'),
 ('CDLME007', 'Insured Group or Policy Number', '80'),
 ('CDLME009', 'Medicaid AID Category ', '50'),
 ('CDLME011', 'Plan Specific Contract Number', '60'),
 ('CDLME012', 'Subscriber Last Name ', '100'),
 ('CDLME013', 'Subscriber First Name ', '100'),
 ('CDLME015', 'Sequence Number ', '100'),
 ('CDLME017', 'Individual Relationship\nCode', '90'),
 ('CDLME018', 'Member Gender ', '90'),
 ('CDLME019', 'Member Date of Birth ', '90'),
 ('CDLME020', 'Member Last Name ', '100'),
 ('CDLME021', 'Member First Name ', '100'),
 ('CDLME023', 'Member Street Address', '60'),
 ('CDLME024', 'Member City Name ', '60'),
 ('CDLME025', 'Member State or Province', '80'),
 ('CDLME026', 'Member ZIP Code ', '80'),
 ('CDLME036', 'Medical Coverage\nUnder This Plan', '90'),
 ('CDLME037', 'Pharmacy Coverage\nUnder This Plan', '90'),
 ('CDLME039', 'Behavioral Health\nCoverage Under This\nPlan', '90'),
 ('CDLME040', 'Primary Insurance Indicator', '100'),
 ('CDLME041', 'Coverage Type ', '50'),
 ('CDLME045', 'Group Name ', '60'),
 ('CDLME050', 'Plan Effective Date ', '90'),
 ('CDLME061', 'Carrier Specific Unique\nMember ID', '95'),
 ('CDLME062', 'Carrier Specific Unique\nSubscriber ID', '95'),
 ('CDLME064', 'High Deductible Plan Indicator', '50'),
 ('CDLME899', 'Record Type ', '100')]

</details>

2. Now add the following snippet to the same file on line 38

<details>

submitters = [ (2, 'TESTGOLD', 10000001, 'gmunoz1', 'CHCD'), (3, 'TESTGOLD', 10000002, 'gmunoz1', 'CHCD'), (1, 'TESTGOLD', 10000000, 'gmunoz1', 'CHCD') ]
</details>

3. Go to [http://localhost:8000/submissions/threshold-exception/](http://localhost:8000/submissions/threshold-exception/)
4. Try adding and removing block and make sure the data populates dynamically
5. Try submitting and make sure submission is successful

## UI

Screenshot 2023-06-13 at 5 27 42 PM

Screenshot 2023-06-13 at 5 28 01 PM

Dynamic field demonstration:

Screenshot 2023-06-13 at 5 28 30 PM

### CODE REVIEW

**Review by happycodemonkey — APPROVED** [review]
This looks great!

File: apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html [hunk 1]
```diff
@@ -294,6 +311,8 @@ <h4>Acknowledgment of Terms</h4>
                     <div id="help-text-threshold-requested_${exceptions}" class="help-text">
                     </div>
                   </div>
+                  </div>
+                  </div>
                   </div>`
```
Comment (edmondsgarrett) [inline apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html:316]: Are all of these needed? Not sure if I'm missing the opening `<div>` tags for these.

**Review by edmondsgarrett — CHANGES_REQUESTED** [review]
Page is looking good, working as expected! Just left a couple of comments to look at.

File: apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html (hunk 1 above)
Comment (sophia-massie) [reply apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html:316]: Good catch - removing the last two div

**Review by edmondsgarrett — APPROVED** [review]
Thanks for the changes, LGTM!

**Review by fnets — APPROVED** [review]
LGTM!

**Pull request merged**